src/app/tracing/dialog/station-properties/station-properties.component.html

```diff
@@ -1,4 +1,17 @@
-<div mat-dialog-content>
+<div
+    mat-dialog-content
+    cdkDrag
+    cdkDragRootElement=".cdk-overlay-pane"
+    cdkDragBoundary=".cdk-overlay-container"
+>
+    <div class="example-handle">
```

Comment:
Please use `fcl-` prefix for custom class names. So our class names can be detected easily.

Reply:
👍